docs/governance/ACCESS_CONTROL_POLICY.md

```
@@ -0,0 +1,429 @@
+# Access Control Policy
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)